frontend/src/js/directives/directives.js

```diff
@@ -19,43 +14,84 @@
         return directive;
 
         function link(scope, element, attrs) {
-
             function headerComp() {
                 var self = this;
                 this.init();
             };
-
             headerComp.prototype = {
                 init: function() {
                     var self = this;
-
                     // initialized mobile sidebar
                     angular.element(".button-collapse").sideNav({
                         menuWidth: 200,
                         closeOnClick: true,
                         draggable: true
                     });
-
                 }
             };
-
             var headerObj = new headerComp();
         }
     }
-
 })();
+//Landing Footer directive
+(function() {
+    'use strict'
```

Comment:
add semicolon after `use strict`. otherwise missing semicolon will cause problem on production since this file will be be compressed.